engine/src/main/java/com/redhat/cloud/notifications/recipients/itservice/ITUserServiceWrapper.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.cloud.notifications.recipients.itservice;
+
+import com.redhat.cloud.notifications.recipients.itservice.pojo.request.ITUserRequest;
+import com.redhat.cloud.notifications.recipients.itservice.pojo.response.ITUserResponse;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+
+@ApplicationScoped
+public class ITUserServiceWrapper {
```

Comment:
I already asked that in the previous PR: why do we need a wrapper?

Reply:
I didn't manage to get it up and running with two RestClients in RbacRecipientUsersProvider. In the mid term we will remove the other rest client and then we can also remove the wrapper.